website/scripts/send-data-to-vanta.js

```diff
@@ -198,6 +198,10 @@ module.exports = {
           return new Error(`When sending a request to the Fleet instance's /hosts/${host.id} endpoint for a Vanta connection (id: ${connectionIdAsString}), an error occurred: ${err}`);
         });
 
+        if(!detailedInformationAboutThisHost.host) {
+          throw new Error(`When sending a request to the Fleet instance's /hosts/${host.id} endpoint for a Vanta connection (id: ${connectionIdAsString}), the response from the Fleet API did not include a host. Response from the Fleet API:`, detailedInformationAboutThisHost);
```

Comment:
Error constructor only takes one argument

Reply:
👍 I updated these errors to `util.inspect()` the response from the Fleet API